Gruntfile.coffee

```diff
@@ -1329,6 +1348,15 @@ module.exports = (grunt) ->
 				options:
 					testname: "Local Test - <%= grunt.template.today('yyyy-mm-dd hh:MM') %>"
 
+		"string-replace":
+			inline:
+				files:
+					'dist/wet-boew/js/': 'dist/wet-boew/js/*.js'
```

Comment:
Could target the `concat.core.dest` and `concat.coreIE8.dest` values to glob through less files.

Reply:
@nschonni is this the last thing I need to resolve?